orcid-scheduler-web/src/main/resources/log4j.xml

```diff
@@ -12,7 +12,7 @@
 	</appender>
 
 	<appender name="CONSOLE" class="org.apache.log4j.ConsoleAppender">
-		<param name="Threshold" value="WARN"/>
+		<param name="Threshold" value="INFO"/>
```

Comment:
Hey @bobcaprice Why you changed this? Wouldn't this generate a lot of unnecessary logging once it gets prod?

Reply:
@bobcaprice could you please roll this back? Thanks